src/lqcut.jl

```diff
@@ -109,8 +109,8 @@ function lq_ctc_correction(
     pol_fit_func = report_µ.f_fit
 
     # property function for drift time correction
-    lq_class_func = "$lq_e_corr_expression - " * join(["$(par[i]) * ($dt_eff_expression)^$(i-1)" for i in eachindex(par)], " - ")
-    lq_class_func_generic = "lq / e  - (slope * qdrift / e + y_inter)"
+    lq_class_func = "( $lq_e_corr_expression ) - " * join(["$(par[i]) * ($dt_eff_expression)^$(i-1)" for i in eachindex(par)], " - ")
```

Comment:
Not sure if this is an overkill, but it doesn't harm to add a set of brackets around the whole expression here:
```suggestion
    lq_class_func = "( ( $lq_e_corr_expression ) - " * join(["$(par[i]) * ($dt_eff_expression)^$(i-1)" for i in eachindex(par)], " - ") * " )"
```

Reply:
Mhm, they should be overkill, because i added in the `lq_norm` function brackets around the `lq_class_expression`:

`lq_norm_func = " ( ($(lq_class_expression))  - $(mvalue(fit_result.μ)) ) / ( $(mvalue(fit_result.σ)) )"`

So it think it would be overkill to have both brackets